src/main/java/br/com/budgetboys/pf_finance_service/domain/income/IncomeRepository.java

```diff
@@ -6,6 +6,6 @@
 public interface IncomeRepository {
     Income save(Income income);
     Income findById(UUID id);
-    List<Income> findAll();
+    List<IncomeResponseDTO> findAll();
```

Comment:
Aqui o correto não seria o Income, mesmo não?
```suggestion
    List<Income> findAll();
```